tests/unit/htdocs/class/file/XoopsFileTest.php

```
@@ -0,0 +1,321 @@
+    }
+
+    /**
+     * getHandler() with no arguments defaults to 'file' handler and returns XoopsFileHandler.
+     */
+    public function testGetHandlerDefaultReturnsXoopsFileHandler(): void
+    {
+        $tmpFile = tempnam(sys_get_temp_dir(), 'xoops_test_');
+        $this->tempFiles[] = $tmpFile;
+
+        // Default $name is 'file', but we still need a valid path
+        $handler = XoopsFile::getHandler('file', $tmpFile);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\XoopsFile' in method 'testGetHandlerDefaultReturnsXoopsFileHandler'. (undefined)

(StaticAccess)